packages/esm-patient-test-results-app/src/print-modal/print-modal.component.tsx

```diff
@@ -0,0 +1,233 @@
+          return t('female', 'Female');
+        case 'other':
+          return t('other', 'Other');
+        case 'unknown':
+          return t('unknown', 'Unknown');
+        default:
+          return gender;
+      }
+    };
+
+    const identifiers =
+      patient?.patient?.identifier?.filter(
+        (identifier) => !excludePatientIdentifierCodeTypes?.uuids.includes(identifier.type.coding[0].code),
+      ) ?? [];
+
+    return {
+      name: `${patient?.patient?.name?.[0]?.given?.join(' ')} ${patient?.patient?.name?.[0].family}`,
+      age: age(patient?.patient?.birthDate),
+      gender: getGender(patient?.patient?.gender),
+      location: patient?.patient?.address?.[0].city,
+      identifiers: identifiers?.length ? identifiers.map(({ value, type }) => value) : [],
+    };
+  }, [patient, t, excludePatientIdentifierCodeTypes?.uuids]);
+
+  const testResults = useMemo(() => {
+    if (selectedFromDate && selectedToDate) {
+      const selectedDateObj = new Date(selectedFromDate);
+      const selectedToDateObj = new Date(selectedToDate);
+
+      return panels
+        .filter((panel) => {
+          const panelDate = new Date(panel.effectiveDateTime);
+          return panelDate >= selectedDateObj && panelDate <= selectedToDateObj;
+        })
+        .map((panel) => {
+          const panelDate = new Date(panel.effectiveDateTime);
+          const formattedPanelDate = panelDate.toLocaleDateString();
+          return {
+            id: panel.id,
+            testType: panel.name,
+            date: formattedPanelDate,
+            result: panel.value,
+            normalRange: panel?.meta?.range ?? '--',
+          };
+        });
+    }
+
+    return panels.map((panel) => {
+      const panelDate = new Date(panel.effectiveDateTime);
+      const formattedPanelDate = panelDate.toLocaleDateString();
+      return {
+        id: panel.id,
+        testType: panel.name,
+        date: formattedPanelDate,
+        result: panel.value,
+        normalRange: panel?.meta?.range ?? '--',
+      };
+    });
+  }, [panels, selectedFromDate, selectedToDate]);
+  if (!testResults?.length) {
+    return <EmptyState displayText={displayText} headerTitle={headerTitle} />;
+  }
+
+  return (
+    <div>
+      <ModalHeader closeModal={closeDialog} title={t('printTestResults', 'Print test results')} />
+      <ModalBody style={{ maxHeight: '500px', overflowY: 'auto' }}>
+        <div className={styles.datePickers}>
+          <div className={styles.dates}>
+            <span className={styles.datePickerLabel}> {t('from', 'From')}:</span>
+            <DatePicker id="onsetDate" datePickerType="single" dateFormat="d/m/Y" light={isTablet} onChange={date => setSelectedFromDate(date)}>
+              <DatePickerInput
+                size="sm"
+                style={{
+                  width: '100%',
+                  border: '1px solid #ccc',
+                  padding: '0.5rem',
+                  displayText: 'centre',
+                }}
```

Comment:
Styles go in SCSS file, but also, why are you styling this? The default should be correct.